packages/dialtone-css/lib/build/less/components/emoji-text-wrapper.less

```diff
@@ -1,5 +1,16 @@
 .d-emoji-text-wrapper {
-  .d-icon {
-    vertical-align: top;
+  &__text {
+    vertical-align: middle;
+  }
+
+  .d-emoji {
+    position: relative;
+    vertical-align: middle;
+
+    .d-icon {
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
+    }
```

Comment:
absolutely crazy this is what we have to do to center 😆

Reply:
Yeah, centering a changing height image with a text it's more complex that it seemed at first sight. But I think this is the safest way to do it even if the text size change.